src/chunk_tag_t_keywords.h

```diff
@@ -0,0 +1,310 @@
+static chunk_tag_t keywords[] =
+{
+   // TODO: it might be useful if users could add their custom keywords to this list
```

Comment:
As an idea, the user coulf specify similar entries in the config file. Those would have to be added to this table at run time. The table would have to be changed from a static array to a vector.

Reply:
mmmh